components/credit-card/CreditDebitInfo.tsx

```diff
@@ -23,25 +49,43 @@ const CreditDebitInfo = () => {
                 <div className="grid grid-cols-2 gap-11">
                     <div className="w-full bg-purple-200 p-4">
                         <h2 className="text-xl mb-1">Credit Cards</h2>
-                        <p>Allow consumers to borrow money from a financial institution and pay it back later</p>
+                        <div>
+                            Allow consumers to {whiteSpace()} money from a financial institution and {whiteSpace()} it back later
+                        </div>
                     </div>
                     <div className="w-full bg-green-200 p-4">
                         <h2 className="text-xl mb-1">Debit Cards</h2>
-                        <p>Allow consumers to spend money they have deposited in their bank account</p>
+                        <div>
+                            Allow consumers to {whiteSpace()} money they have {whiteSpace()} in their bank account
+                        </div>
                     </div>
                 </div>
             </div>
 
-            <div className="flex flex-row justify-evenly">
-                <h2 className="text-2xl">How are they the same?</h2>
-                <div className=" bg-gray-200 p-4">
-                    <ul className="flex flex-col gap-4">
-                        <li>Allow a person to make purchases</li>
-                        <li>If card is lost or stolen, report import it immediately</li>
-                        <li>A hard plastic card with information stored on a magnetic strip.</li>
-                        <li>Owner's name and an expiration date on the card</li>
-                    </ul>
+            <div className="grid grid-cols-2">
+                <div>
+                    <h3 className="text-2xl mb-4">Word Bank</h3>
+                    <div className="grid grid-cols-3 gap-4 pr-6">
+                        {WordBankData.map((item) => (
+                        <div key={item.id} className="bg-gray-200 flex justify-center">
+                            <span className="m-5 p-3 text-xl" id={item.word} draggable="true" onDragStart={drag}>{item.word}</span>
+                        </div>
+                        ))}
+                    </div>
+                </div>
+
+                <div className="pl-6">
+                    <h2 className="text-2xl mb-4">How are they the same?</h2>
+                    <div className=" bg-gray-200 p-4">
+                        <ul className="flex flex-col gap-4">
+                            <li>Allow a person to make {whiteSpace()}</li>
```

Comment:
I'm not down to merge this in as is. Some ideas:
- change this to be a text input that a student has to type in
- change this to a dropdown where the student has to select from a list of options
- give me some time to explore a re-usable drag and drop component

Reply:
changed to a dropdown with 2 options. Having issues trying to iterate the options from an array